Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -7517,7 +7517,7 @@ Function Set-PinnedApplication {
 				Throw "Action [$Action] not supported. Supported actions are [$($Verbs.Keys -join ', ')]."
 			}
 
```

Comment:
Please add a comment on why we're not using -in but instead .Contains() so the devs later on know it.